pkg/internal/third_party/go-json-experiment/json/testdata_test.go

```diff
@@ -7,7 +7,8 @@ package json
 import (
 	"bytes"
 	"compress/gzip"
-	"io/ioutil"
+	"io"
```

Comment:
We import this from https://github.com/go-json-experiment/json. Can you update this in the upstream repo instead?